scripts/zappa_manage.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+import click
+from pybase64 import b64encode
+
+from asym_crypto_yaml import (
+    load, Encrypted, decrypt_value,
+    load_private_key_from_file,
+    load_private_key_from_string
+)
+
+from zappa_manage import deploy_lambda_envs
```

Comment:
There is only 1 click command however I am unsure if whether or not I should import the other functions given how this [blob](https://github.com/edx/asym-crypto-yaml/blob/master/scripts/asym_crypto_yaml.py#L4-L9) in asym_crypto_yaml imports (almost) everything but doesn't use all

Reply:
Just import what you need IMO

Reply:
Thanks!